code/modules/mob/living/carbon/xenomorph/Abilities.dm

```diff
@@ -453,23 +453,21 @@
 	max_range = 7
 
 /datum/action/xeno_action/activable/salvage_plasma
-	name = "Salvage Plasma"
+	name = "Salvage Essence"
 	action_icon_state = "salvage_plasma"
 	ability_name = "salvage plasma"
-	var/plasma_salvage_amount = PLASMA_SALVAGE_AMOUNT
 	var/salvage_delay = 5 SECONDS
 	var/max_range = 1
 
 datum/action/xeno_action/activable/salvage_plasma/use_ability(atom/A)
 	var/mob/living/carbon/Xenomorph/X = owner
 	if(owner.action_busy)
 		return
-	X.xeno_salvage_plasma(A, plasma_salvage_amount, salvage_delay, max_range)
+	X.xeno_salvage_plasma(A, salvage_delay, max_range)
 
 datum/action/xeno_action/activable/salvage_plasma/improved
-	plasma_salvage_amount = PLASMA_SALVAGE_AMOUNT * 2
 	salvage_delay = 3 SECONDS
-	max_range = 4
+	max_range = 1
```

Comment:
If you're changing it to 1 you can just remove it as the variable is set to 1 a few lines up already.

Reply:
I'd prefer to keep this in the event I feel it could use a range tweak

Reply:
Please re-add it in such event, to avoid unused features bloat.